history/taskana-simplehistory-provider/src/test/java/acceptance/query/QueryClassificationHistoryAccTest.java

```diff
@@ -96,17 +98,31 @@ void should_ReturnEmptyList_When_ProvidingWrongContraints() {
   @Test
   void should_ReturnSingleHistoryEvent_When_UsingSingleMethod() {
     ClassificationHistoryEvent single =
-        historyService.createClassificationHistoryQuery().userIdIn("peter").single();
+        historyService
+            .createClassificationHistoryQuery()
+            .userIdIn("peter")
+            .classificationIdIn("CLI:000000000000000000000000000000000001")
+            .single();
     assertThat(single.getEventType()).isEqualTo(ClassificationHistoryEventType.CREATED.getName());
 
     single =
         historyService
             .createClassificationHistoryQuery()
             .eventTypeIn(ClassificationHistoryEventType.CREATED.getName(), "xy")
+            .classificationIdIn("CLI:000000000000000000000000000000000001")
             .single();
     assertThat(single.getUserId()).isEqualTo("peter");
   }
 
+  @Test
+  void should_ThrowException_When_SingleMethodRetrievesMoreThanOneEventFromDatabase() {
+
+    ClassificationHistoryQuery query =
+        getHistoryService().createClassificationHistoryQuery().userIdIn("peter");
+
+    assertThatThrownBy(() -> query.single()).isInstanceOf(TooManyResultsException.class);
```

Comment:
I think, we should document this Exception in the JavaDoc of the API, shouldn't we?